### PR TITLE
Add strategy VeryBad

### PR DESCRIPTION
**Pull request by janga1997**
#379 
Here is the [paper](http://abricom.org.br/wp-content/uploads/2016/03/bricsccicbic2013_submission_202.pdf) . Page 3.

### CODE REVIEW

File: axelrod/strategies/verybad.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    @staticmethod
+    def strategy(opponent: Player) -> Action:
+        
+        if len(opponent.history) < 4:
+            return C
+        
+        total_moves = len(opponent.history)
+        cooperations = opponent.cooperations
+        
+        cooperation_probability = cooperations / (total_moves * 1.0)
```
Comment (drvinceknight) [inline axelrod/strategies/verybad.py]: There is no need for `* 1.0` here (this is all Python 3).

Comment (janga1997) [inline axelrod/strategies/verybad.py]: Just though it couldn't hurt. Changing it now.

**Comment by drvinceknight** [comment]
I've taken a look at the paper and I'm really not sure there's enough details there for us to assume this is an implementation of that strategy:

> It cooperates in the first three rounds, and uses probability (it implements a memory, which stores the opponent’s moves) to decide for cooperating or defecting.

The fact that they say "uses probability" makes me think that there should be some randomness somewhere...

That doesn't mean this strategy shouldn't be added to the library but I'm not sure we can say it's an implementation of the one from that paper... It might need to change name. Will be good to get the others' thoughts on this.

**Comment by janga1997** [comment]
@drvinceknight I was confused by the lack of details too, but I assumed the most general meaning.
I could definitely be wrong about the intentions of the paper.

**Comment by drvinceknight** [comment]
> I was confused by the lack of details too, but I assumed the most general meaning.
> I could definitely be wrong about the intentions of the paper.

Yeah it's really confusing and one of the problems with modern research where papers are written that are not possible to replicate... It might be that like I said on #866 we might need to say "losely based on a description given in...". Another option might be getting in touch with the authors and explaining what you're doing and asking for more details?

**Comment by janga1997** [comment]
@drvinceknight I will try to contact one of the authors to clarify both.

**Comment by marcharper** [comment]
If we don't hear back from the others just use your best judgement and update the docstrings to make it clear that a complete description could not be obtained.

**Comment by janga1997** [comment]
It has been almost a week, and I haven't heard back from any of the authors.
I'm updating the docstrings for both the strategies.

**Comment by marcharper** [comment]
Thanks @janga1997 . Please check the docstring formatting, that's why the build "failed".

File: axelrod/strategies/verybad.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    """
+    It cooperates in the first three rounds, and uses probability
+    (it implements a memory, which stores the opponent’s moves) to decide for
+    cooperating or defecting.
+    Due to a lack of information as to what that probability refers to in this
+    context, probability(P(X)) refers to (Count(X)/Total_Moves) in this
+    implementation
+    P(C) = Cooperations / Total_Moves
+    P(D) = Defections / Total_Moves = 1 - P(C)
+    Names:
```
Comment (drvinceknight) [inline axelrod/strategies/verybad.py]: Could you add a blank line before `Names` please.

**Pull request merged**